packages/protocol/test/governance/network/governance.ts

```diff
@@ -4187,6 +4187,8 @@ contract('Governance', (accounts: string[]) => {
     })
```

Comment:
Maybe hold this until Governance tests to Foundry get merged? Otherwise I fear these tests may get lost in the process.

Reply:
Done